README.md

```diff
@@ -21,5 +21,114 @@ yarn run unit
 # run all tests
 yarn test
 ```
+## Data model
```

Comment:
I would move this to the top, and move Build Setup to the bottom (of everything)